.ci/update-vendoring.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will allow to vendor kata-containers/runtime
```

Comment:
if you want to test `kata-containers/runtime/virtcontainers` in `clearcontainers/runtime`, you can

```
cd $GOPATH/src/github.com/clearcontainers/runtime/vendor/github.com/kata-containers
rm -rf runtime
git clone https://${AUTHOR_REPO_GIT_URL}
pushd runtime
git checkout ${COMMIT_REVISION}
[ -d "virtcontainers/vendor" ] && rm -rf virtcontainers/vendor
[ -d "vendor" ] && rm -rf vendor
popd
```